src/BabylonCpp/src/core/system.cpp

```diff
@@ -51,7 +51,7 @@ void openBrowser(const std::string& url)
   std::string jsCode = "window.open('url','_blank');";
   jsCode             = StringTools::replace(jsCode, "url", url);
   emscripten_run_script(jsCode.c_str());
-#elif defined(__APPLE__)
+#elif defined(__APPLE__) && defined(__x86_64__)
```

Comment:
So it turns out the __MACH__ is also defined when configuring the iOS target. Maybe there's something more to do in cmake to get this correct but I did not see what. My solution which works for now was to add a check for __x86_64__ along with __APPLE__ and that does the trick. Apple's documentation was not helpful https://developer.apple.com/library/archive/documentation/Porting/Conceptual/PortingUnix/compiling/compiling.html